l10n_ec_ote/README.rst

```diff
@@ -0,0 +1,33 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Bug Tracker
```

Comment:
Hi, I tried to enter to the runbot site but it gives an error:
Redirects to.
http://runbot.odoo-community.org:8976/login?db=3135654-9-0-345adc-all&login=admin&key=admin&redirect=/web?debug=1

Error:
Firefox can't establish a connection to the server at runbot.odoo-community.org:8976.
    The site could be temporarily unavailable or too busy. Try again in a few moments.